pages/configuration/graylog_ctl.rst

```diff
@@ -238,6 +238,9 @@ Choose the Graylog version you want to install from the `list of Omnibus package
   $ sudo graylog-ctl stop
   $ sudo dpkg -G -i graylog_latest.deb
   $ sudo graylog-ctl reconfigure
+  $ sudo reboot
+
+.. note:: Restart the Server after you had updated should be done to prevent data corruption!
```

Comment:
`Reboot the server after the update procedure to make sure that all services are running in the correct version.` - data corruption sounds like Elasticsearch will break otherwise what is not the case.